certora/specs/RyskStrategy.spec

```diff
@@ -0,0 +1,217 @@
+import "erc20.spec"
+
+using MockLUSD as underlying
+using MockRyskLiquidityPool as ryskLqPool
+using Vault as vault
+
+methods {
+
+    // admin/manager functions
+    transferAdminRights(address)
+    invest()
+    completeWithdrawal()
+    withdrawToVault(uint256)
+    transferYield(address, uint256)
+
+    // view functions
+    investedAssets() returns (uint256) envfree
+    hasAssets() returns (bool) envfree
+    isSync() returns (bool) envfree
+    hasRole(bytes32, address) returns (bool) envfree
+
+    // public variables
+    vault() returns (address) envfree
+    ryskLqPool() returns (address) envfree
+    MANAGER_ROLE() returns (bytes32) envfree
+    KEEPER_ROLE() returns (bytes32) envfree
+    SETTINGS_ROLE() returns (bytes32) envfree
+    DEFAULT_ADMIN_ROLE() returns (bytes32) envfree
+
+    // erc20
+    underlying.balanceOf(address) returns (uint256) envfree
+}
+
+definition adminFunctions(method f) returns bool =
+    f.selector == transferAdminRights(address).selector;
+
+definition managerFunctions(method f) returns bool =
+    f.selector == invest().selector
+    ||
+    f.selector == withdrawToVault(uint256).selector
+    ||
+    f.selector == transferYield(address, uint256).selector; // There is an issue here
```

Comment:
`transferYield` is inherited from `BaseStrategy` where it is not a manager function, hence it should be removed here